module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Add-EntraAdministrativeUnitMember.md

```diff
@@ -33,14 +33,20 @@ Add-EntraAdministrativeUnitMember
 ```
 
 ## Description
-The **Add-EntraAdministrativeUnitMember** cmdlet adds a Microsoft Entra ID administrative unit member.
+
+The `Add-EntraAdministrativeUnitMember` cmdlet adds a Microsoft Entra ID administrative unit member.
 
 ## Examples
 
 ### Example 1: Add user as an administrative unit member
 
 ```powershell
-PS C:\>Add-EntraAdministrativeUnitMember -Id f306a126-cf2e-439d-b20f-95ce4bcb7ffa -RefObjectId d6873b36-81d6-4c5e-bec0-9e3ca2c86846
+Connect-Entra -Scopes 'AdministrativeUnit.ReadWrite.All'
+$params = @{
+    Id = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```

Comment:
```suggestion
    ObjectId = 'aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb'
```

Reply:
Replace `Id` with `ObjectId` in all instances

Reply:
fixed and updated